pkg/omf/cli/omf.install_package.fish

```diff
@@ -1,5 +1,12 @@
-function omf.install_package
+# Return success for at least one successful install.
+function omf.install_package 
+  set -l return_code $OMF_UNKNOWN_ERR
   for search in $argv
     omf.install --pkg $search
+    set -l install_code $status
+    if test $install_code -eq 0
```

Comment:
At first read, this make theme pkg mandatory in case of name collision.
I don't know if it is a expected behaviour.

It does give me some idea for the organization of my code.